lib/manager/nuget/util.ts

```diff
@@ -25,7 +25,16 @@ export function getRandomString(): string {
   return cryptoRandomString({ length: 16 });
 }
 
-export async function determineRegistries(
+export function getDefaultRegistries(): Registry[] {
+  return datasourceNuget.defaultRegistryUrls.map(
+    (registryUrl) =>
+      ({
+        url: registryUrl,
+      } as Registry)
+  );
+}
+
+export async function getConfiguredRegistries(
```

Comment:
I renamed `determineRegistries` to `getConfiguredRegistries` to make it clearer that this just returns something if there is a NuGet config. It's a bit weird, as actually I would have expected this to return the default registries if there is no config file, but I think we did it for back-compatibility back then.